src/garage/envs/point_env.py

```diff
@@ -79,6 +79,7 @@ def step(self, action):
         a = action.copy()  # NOTE: we MUST copy the action before modifying it
         a = np.clip(a, self.action_space.low, self.action_space.high)
 
+        self._point = np.clip(self._point + a, -5, 5)
```

Comment:
perhaps define `ARENA_SIZE = 5` at the top of the file to make for easier maintenance.